helm_deploy/values-prod.yaml

```diff
@@ -14,11 +14,17 @@ generic-service:
     API_BASE_URL_MAPPING: https://nomis-sync-prisoner-mapping.hmpps.service.justice.gov.uk
     API_BASE_URL_VISITS: https://visit-scheduler.prison.service.justice.gov.uk
     API_BASE_URL_INCENTIVES: https://incentives-api.hmpps.service.justice.gov.uk
-    API_BASE_URL_ACTIVITIES: https://activities-api.prison.service.justice.gov.uk
+    API_BASE_URL_ACTIVITIES: https://activities-api-dev.prison.service.justice.gov.uk
```

Comment:
this is fine, but recently we just point back at this service, it just needs to have a health point

Reply:
ok noted!